### PR TITLE
[IMP] reference/user_interface: update documentation regarding widget in graph

### PR DESCRIPTION
**Pull request by sami-odoo**
This commit aims to update the documentation regarding the adaptability of widgets in 
the graph view of Odoo.

Task-3861721

### CODE REVIEW

File: content/developer/reference/user_interface/view_architectures.rst [hunk 1]
```diff
@@ -2872,8 +2886,8 @@ sorted on the string of the field.
     which means that non-stored function fields can not be used in pivot views
 
 
-In Pivot view a ``field`` can have a ``widget`` attribute to dictate its format.
-The widget should be a field formatter, of which the most interesting are
+In Pivot view, similar to Graph view, a ``field`` can have a ``widget`` attribute to dictate
```
Comment (Polymorphe57) [inline content/developer/reference/user_interface/view_architectures.rst:2889]: no need to modify the part for pivot

File: content/developer/reference/user_interface/view_architectures.rst [hunk 2]
```diff
@@ -2801,6 +2801,20 @@ sorted on the string of the field.
...
+
+In Graph view a ``field`` can have a ``widget`` attribute to dictate its format.
+The widget should be a field formatter, of which the most interesting are
+``date``, ``datetime``, ``float_time``, and ``monetary``.
+
+For instance a task graph view could be defined as::
+
+    <graph string="Tasks">
+        <field name="project_id"/>
+        <field name="stage_id"/>
```
Comment (Polymorphe57) [inline content/developer/reference/user_interface/view_architectures.rst:2813]: remove

File: content/developer/reference/user_interface/view_architectures.rst [hunk 3]
```diff
@@ -2801,6 +2801,20 @@ sorted on the string of the field.
...
+        <field name="working_hours_open" widget="float_time"/>
```
Comment (Polymorphe57) [inline content/developer/reference/user_interface/view_architectures.rst:2814]: remove

File: content/developer/reference/user_interface/view_architectures.rst [hunk 4]
```diff
@@ -2801,6 +2801,15 @@ sorted on the string of the field.
    graph view aggregations are performed on database content, non-stored
    function fields can not be used in graph views
 
+
+In Graph view a ``field`` can have a ``widget`` attribute to dictate its format.
+The widget should be a field formatter, of which the most interesting are
+``date``, ``datetime``, ``float_time``, and ``monetary``.
```
Comment (Polymorphe57) [inline content/developer/reference/user_interface/view_architectures.rst:2807]: you can remove date and datetime here (date/datetime fields are not used as measures)

Comment (sami-odoo) [reply content/developer/reference/user_interface/view_architectures.rst]: Hello 

Changes have been done

File: content/developer/reference/user_interface/view_architectures.rst [hunk 5]
```diff
@@ -2801,6 +2801,15 @@ sorted on the string of the field.
    graph view aggregations are performed on database content, non-stored
    function fields can not be used in graph views
 
+
+In Graph view a ``field`` can have a ``widget`` attribute to dictate its format.
```
Comment (AntoineVDV) [inline content/developer/reference/user_interface/view_architectures.rst:2805]:
```suggestion
In Graph views, a ``field`` can have a ``widget`` attribute to dictate its format.
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+ delegate=Polymorphe57

File: content/developer/reference/user_interface/view_architectures.rst (hunk 5 above)
Comment (sami-odoo) [reply content/developer/reference/user_interface/view_architectures.rst]: i did the changes

Thanks

**Comment by Polymorphe57** [comment]
@robodoo r+

**Pull request closed without merging**